src/network-manager/azext_network_manager/__init__.py

```diff
@@ -7,18 +7,17 @@
 # Changes may cause incorrect behavior and will be lost if the code is
 # regenerated.
 # --------------------------------------------------------------------------
-
+# pylint disable=unused-import
 from azure.cli.core import AzCommandsLoader
-import azext_network_manager._help
+import azext_network_manager._help  # pylint: disable=unused-import
```

Comment:
we already have a global disable for that

Reply:
Seems like it was still raising pylint error. Use this disable to be safe.